corehq/apps/app_manager/templates/app_manager/module_view.html

```diff
@@ -4,7 +4,6 @@
 {% load i18n %}
 {% block js %}{{ block.super }}
     <script src="{% static 'style/ko/knockout_bindings.ko.js' %}"></script>
-    <script src="{% static 'hqwebapp/js/lib/underscore-1.4.4.js' %}"></script>
```

Comment:
@benrudolph Were you just removing this, or does django-bower add this back to the page somehow?